src/main.rs

```diff
@@ -80,15 +76,6 @@ fn main() -> Result<(), aes_gcm_siv::Error> {
         let f_i_bytes = cipher.encrypt(nonce, m_prime_i_string.as_bytes().as_ref())?;
```

Comment:
i suspect there is better way to do this instead of converting BigInt->String->bytes

Reply:
also in case of failure here, this panics since `main()` just propagates the error.

Reply:
The BigInt->String->bytes conversion was just me trying to reduce the error surface as much as possible to finish the rough draft as soon as possible. The examples online used `.encrypt()` with a string turned into bytes as input, so I converted BigInt to String before applying the strategy of the examples. A next step is converting BigInt directly to bytes.

Reply:
Yes, I had to change the signature of `main()` to make the `encrypt()` work, and know that it's probably not good practice. Exploring better ways was low priority.